keyboards/horrortroll/caticorn/rev1/hotswap/keymaps/default/keymap.c

```diff
@@ -23,11 +23,9 @@
 
 enum layer_names {
     _BASE,
-    _FN,
+    _FN
```

Comment:
This should be reverted. 
```suggestion
    _FN,
```
While not critical, clang-format will condense this to one line for the entire enum. The comma at the end prevents it from doing so.